clientv3/balancer.go

```diff
@@ -64,6 +64,23 @@ func newSimpleBalancer(eps []string) *simpleBalancer {
 	return sb
 }
 
+func (b *simpleBalancer) update(eps ...string) {
+	epm := make(map[string]struct{})
```

Comment:
why do we need this map? we can change line 77 to range over the passed in eps, no?